config/locales/models/motif.fr.yml

```
@@ -82,13 +85,30 @@ fr:
       motif/location_types/hint:
         public_office: Le RDV aura lieu au lieu sélectionné (MDS...)
         visio: Nous vous fournirons un lien de visioconférence unique pour chaque RDV. Vous devrez vous connecter au <a href="https://www.numerique.gouv.fr/outils-agents/webconference-etat/" target="_blank">service de webconférence de l'État <i class="fa fa-external-link" /></a> avec Agent Connect pour démarrer la visioconférence.
-        phone: L'usager sera notifié que le RDV se passera par téléphone, au numéro indiqué dans son profil
-        home: L'usager sera notifié que le RDV aura lieu à l'adresse remplie dans son profil
+        phone: L’usager est notifié que le RDV aura lieu par téléphone, au numéro indiqué sur la fiche de l’usager.
```

Historiquement on parle de notification dans ce texte, mais c'est faux car ça dépend des notifications choisies, et c'est pas l'info pertinente.
```suggestion
        phone: L’agent appelle le numéro indiqué sur la fiche de l'usager.
```

---

Décision du trio : on préfère s'aligner sur l'existant.